app/views/admins/user_search.mobile.haml

```diff
@@ -0,0 +1,56 @@
+.col-md-9
+  .row
+    .user_search.col-md-8
+      %h3= t('admins.admin_bar.user_search')
+      = form_for @search, url: {action: 'user_search'}, html: {method: :get, class: 'form-horizontal'} do |f|
+        .form-group
+          = f.label :username, t('username'), class: 'col-sm-2 control-label'
+          .col-sm-10
+            = f.text_field :username, class: "form-control"
+
+        .form-group
+          = f.label :email, t('email'), class: 'col-sm-2 control-label'
+          .col-sm-10
+            = f.text_field :email, class: "form-control"
+
+        .form-group
+          = f.label :guid, t('admins.user_entry.guid'), class: 'col-sm-2 control-label'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.